.github/workflows/bump-sec-scanners-config-reusable.yml

```diff
@@ -87,29 +84,30 @@ jobs:
             echo "CREATE_PR=true" >> $GITHUB_ENV 
           fi
 
-      - name: Print Content of sec-scanners-config.yaml
         if: ${{ always() }}
```

Comment:
Why do we need `if always`?

Reply:
always() will make sure this will get executed even if an earlier step fails. This is particularly useful for debugging say in case you want to know if at least rendering the file went all right.

Reply:
always() will make sure this will get executed even if an earlier step fails. This is particularly useful for debugging say in case you want to know if at least rendering the file went all right.